### DISCUSSION TITLE
关于23版本科毕设word模板与本模板的区别（欢迎补充）

### DISCUSSION THREAD

**Discussion by auto-Dog** [sjtug/SJTUThesis#956] [展示区]
23版本科毕设word版本模板链接：[官方word模板](https://bysj.jwc.sjtu.edu.cn/ShowNews.aspx?NewsNo=dr5WpcF4fpLFKYbXV4YDjg....)

目前已发现的区别是：
- [x] 封面上申请学位，本科生也要写 申请学位：学士，并且大标题是 上海交通大学学位论文 ，而不是 ~~...学士学位论文~~ 解决方案在 [https://github.com/sjtug/SJTUThesis/issues/941](https://github.com/sjtug/SJTUThesis/issues/941)

- [x] 封面中英文日期，中文只写到月，英文写到日期。见[https://github.com/sjtug/SJTUThesis/discussions/964](https://github.com/sjtug/SJTUThesis/discussions/964)

- [ ] 目录问题：word模板要求字号按级别顺序由大到小。目前官方latex模板里也没实现 相关解决：[https://github.com/sjtug/SJTUThesis/discussions/963](https://github.com/sjtug/SJTUThesis/discussions/963)

- [x] 目录问题：word模板编号了摘要和英文摘要，见[https://github.com/sjtug/SJTUThesis/discussions/942](https://github.com/sjtug/SJTUThesis/discussions/942) 这种方法可能导致浏览时导航栏有两个摘要

- [x] 标题问题：“参考文献”标题应该每个字中间空一格， 即 **参\quad 考\quad 文\quad 献**，`\sjtusetup{name/bib={参\quad 考\quad 文\quad 献}}`
- [x] 标题问题：“致谢”标题也应该空格，官方latex给出的是 **致 \qquad 谢** ，`\sjtusetup{name/ack={致 \qquad 谢}}`
- [ ] 封面问题：中文扉页姓名~学院这几行中字间距 [https://github.com/sjtug/SJTUThesis/discussions/960](https://github.com/sjtug/SJTUThesis/discussions/960)

- [x] 图，表的索引页问题：目前word模版中**没有**。而且部分同学接到图书馆反馈也不应该放在文章前面，可在 `main.tex` 内调整 `\listoffigures*`, `\listoftables*`, `\listofalgorithms*`
- [x] 多个页眉问题：word模板中没有中英文摘要，大摘要的名称页眉 解决方案包括[https://github.com/sjtug/SJTUThesis/discussions/922](https://github.com/sjtug/SJTUThesis/discussions/922) 以及 [https://github.com/sjtug/SJTUThesis/discussions/953](https://github.com/sjtug/SJTUThesis/discussions/953)

**Comment by encounter5639** [comment]
根据图书馆审核反馈意见，插图、插表目录位置应置于参考文献之后

**Reply by encounter5639** [reply]
在main.tex中调整\listoffigures*、\listoftables*位置

**Comment by encounter5639** [comment]
论文奇数页、偶数页左侧页边距似乎不一致，图书馆审核反馈需调整

**Reply by LogCreative** [reply]
最好使用 `oneside` 而不是 `openany` 选项。

**Reply by encounter5639** [reply]
在main.tex中修改\documentclass[type=bachelor]{sjtuthesis}为\documentclass[type=bachelor,oneside]{sjtuthesis}

**Comment by Erich-Chen** [comment]
请问如何在“摘要”和 “Abstract" 前分别添加论文题目？要求如下：

- 中文论文标题，1行或2行，黑体，加粗，三号，居中；空一行；后接“摘\quad要”标题。
- 论文英文标题：Times New Roman，14磅，加粗，居中；空一行；后接 "ABSTRACT" 标题。

个人理解大约要修改 sjtuthesis.cls 的 Line 1220 附近，但我不能准确理解这里的配置语法。

谢谢！
